pkgs/desktops/gnome/extensions/guillotine/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-guillotine";
+  version = "24";
+
+  src = fetchFromGitLab {
+    owner = "ente76";
+    repo = "guillotine";
+    rev = "v${version}";
+    hash = "sha256-eNhK3h9luPGXHR3lPkfu/mUN9+ixma64rbCk0cjF4Fc=";
+  };
+
+  dontBuild = true;
+
+  passthru = {
+    extensionUuid = "guillotine@fopdoodle.net";
+    extensionPortalSlug = "guillotine";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/gnome-shell/extensions/guillotine@fopdoodle.net
+    cp -R schemas "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
```

Comment:
`gschemas.compiled` file is platform-specific so you need to remove it and recompile it using `glib-compile-schemas`.